constellation_orderboard/views.py

```diff
@@ -52,23 +57,17 @@ def view_board(request, board_id):
     newForm = CardForm()
     editForm = CardForm(prefix="edit")
 
-    can_move = board_perms(request.user, 'move', board_id)
-    can_add = board_perms(request.user, 'add', board_id)
-    can_delete = board_perms(request.user, 'delete', board_id)
-
     return render(request, 'constellation_orderboard/board.html', {
         'form': newForm,
         'editForm': editForm,
         'id': board_id,
         'template_settings': template_settings,
-        'can_move': can_move,
```

Comment:
Lets add these back so that we can do switching in the template renderer based on what the user can actually do.